pages/common/fastboot.md

```diff
@@ -26,3 +26,7 @@
 - Display connected devices:
 
 `fastboot devices`
+
+- Display all device info:
```

Comment:
IMO this might be a little dubious. 
Right before this example, you use ``devices`. This might lead for people to think that this command will display all information for all devices.
I would recommend changing this to something that clearly states that this is displaying the information from a single device.
```suggestion
- Display all information of a device:
```